cmd/mailer/Dockerfile

```diff
@@ -24,4 +24,4 @@ WORKDIR /app
 COPY --from=build /app/mailer .
 
 # Set the entrypoint command
-ENTRYPOINT ["/app/mailer"]
+ENTRYPOINT ["/app/mailer", "--scheduler-interval", "15s"]
```

Comment:
IMHO sending messages every 15 seconds is to frequent)

Reply:
Forgot to remove after email testing, fixing it